herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_array/IX_detector_array.m

```diff
@@ -22,6 +22,8 @@
     properties (Dependent)
         % Detector identifiers, unique integers greater or equal to one
         id
```

Comment:
very incomplete implementation of a serializable class. 
1) no checks on setters. Apparently it accepts only detector banks, but at current form would accept anything. 
2) is setting sigle bank deletes all current banks or merges them?
3) If a serializable class is defined by its properties, why not to use standard serializable constructor? If it is not defined by the properties, its not normal serializable class.

Reply:
After discussion with @abuts it is clear that the interface for this is ill-defined. For 1 bank per detector_array, it behaves somewhat like a bank for get methods, but only sets a complete bank. For more than one bank, it concatenates each bank field into a single field for the whole array. We need a description of how this will be used before we can proceed.